docs/sources/style-guide/style-conventions/index.md

```diff
@@ -58,6 +58,7 @@ Simple, direct communication is the key to effective technical communication.
 - Don't use buzzwords or jargon.
 - Keep paragraphs to three sentences or less.
   - Condense the text, add more headings, or do both.
+- Use contractions in most cases to express an informal style.
```

Comment:
```suggestion
- Use contractions in common and negative cases: _you’re_, _that’s_, _isn’t_ or _don’t, for example.
```
I prefer to give more specific guidance than _most_.

Reply:
The style has been more _conversational_ than _informal_, because we advice folks to "Write as though you are having a conversation about technology with a friend."